src/pages/ClassPageCenterTable.vue

```diff
@@ -154,23 +211,17 @@ export default {
 
       // `tableAssignments` has the structure of: [{ roomID: "123", "assignees": ["345", "abc"] }]
       let i = 0; 
-      const groupSize = 3; 
       for (const student of connectedStudents) {
-        if (tableAssignments[i].assignees.length > groupSize) {
-          i += 1; 
+        if (tableAssignments[i].assignees.length >= this.groupSizeForRandom) {
```

Comment:
I don't understand this logic: could you explain with a quick video?

Reply:
https://explain.mit.edu/class/mDbUrvjy4pe8Q5s5wyoD/posts/sNf9EAfORtZmiBjLYnts